### PR TITLE
PIMS-662 - Fix Issues in Analytics

### PR DESCRIPTION
**Pull request by LawrenceLau2020**
## Description

Fixing the remaining data issues for Dan.

[](https://hamzamohdzubair.github.io/redant/)

- [PIMS-662](https://apps.itsm.gov.bc.ca/jira/browse/PIMS-662)

[](https://hamzamohdzubair.github.io/redant/)

- None listed.

## Type(s) of Change

- [x] Bug fix
- [ ] Dependency updates, additions, or removal
- [ ] New feature
- [ ] Refactoring code or cleanup
- [ ] Documentation changes or additions
- [ ] Testing changes or additions
- [ ] Design and user experience
- [ ] Optimizations or security

## Checklist:

- I have performed a self-review of my own code.
- I have commented my code, particularly in hard-to-understand areas.
- I have made corresponding changes to the documentation where required.
- My changes generate no new warnings.

- [ ] I have read the checklist.

### CODE REVIEW

File: frontend/src/components/maps/leaflet/Map.tsx [hunk 1]
```diff
@@ -279,8 +279,9 @@ const Map: React.FC<MapProps> = ({
         address: filter.address ?? '',
         pid_pin: filter.pid ?? '',
         property_name: filter.name ?? '',
-        classification:
-          lookupCodes.getClassificationNameById(Number(filter.classificationId)) ?? '',
+        classification: filter.classificationId
+          ? lookupCodes.getClassificationNameById(Number(filter.classificationId))
+          : 'nothing',
```
Comment (BradyMitch) [inline frontend/src/components/maps/leaflet/Map.tsx:284]:
```suggestion
          : '',
```
Empty would probably be better

File: frontend/src/features/projects/list/ProjectListView.tsx [hunk 2]
```diff
@@ -197,7 +197,10 @@ export const ProjectListView: React.FC<IProps> = ({
           schema: 'iglu:ca.bc.gov.pims/search/jsonschema/1-0-0',
           data: {
             view: location.pathname.includes('/spl') ? 'spl_projects' : 'agency_projects',
-            agency: lookupCodes.getAgencyFullNameById(Number(filter.agencies)) ?? '',
+            agency:
+              filter.agencies?.length == 0
```
Comment (BradyMitch) [inline frontend/src/features/projects/list/ProjectListView.tsx:201]:
```suggestion
              filter.agencies?.length === 0
```

File: frontend/src/components/maps/leaflet/Map.tsx (hunk 1 above)
Comment (LawrenceLau2020) [reply frontend/src/components/maps/leaflet/Map.tsx:284]: Thanks for catching that, lol......did a cut and paste and forgot to remove that.

File: frontend/src/components/maps/leaflet/Map.tsx [hunk 3]
```diff
@@ -281,7 +281,7 @@ const Map: React.FC<MapProps> = ({
         property_name: filter.name ?? '',
         classification: filter.classificationId
           ? lookupCodes.getClassificationNameById(Number(filter.classificationId))
-          : 'nothing',
+          : '',
```
Comment (LawrenceLau2020) [inline frontend/src/components/maps/leaflet/Map.tsx:284]: got rid of the debug text.

**Review by BradyMitch — APPROVED** [review]

**Pull request merged**